homeassistant/components/switchbot/strings.json

```diff
@@ -24,8 +24,7 @@
     "step": {
       "init": {
         "data": {
-          "retry_count": "Retry count",
-          "retry_timeout": "Timeout between retries"
+          "retry_count": "Retry count"
```

Comment:
We need to update the options flow too.

Reply:
It works better if I actually save the files before I commit :)

Fixed